app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -19,10 +27,42 @@ class EcsCapacityProvider extends React.Component<IEcsCapacityProviderProps, IEc
     const cmd = this.props.command;
 
     this.state = {
-      capacityProviderStrategy: cmd.capacityProviderStrategy,
+      availableCapacityProviders: this.setAvailableCapacityProviders(cmd),
+      defaultCapacityProviderStrategy: this.setDefaultCapacityProviderStrategy(cmd),
+      ecsClusterName: cmd.ecsClusterName,
+      useDefaultCapacityProviders: cmd.useDefaultCapacityProviders || cmd.capacityProviderStrategy && cmd.capacityProviderStrategy.length == 0,
+      capacityProviderStrategy: cmd.capacityProviderStrategy.length > 0 ? cmd.capacityProviderStrategy : [],
```

Comment:
if capacityProviderStrategy is null, wouldn't this throw an exception?

Reply:
Ahh, yes. Fixed it.